### ISSUE TITLE
Server Crash

### ISSUE THREAD

**Issue by LEclipse1001** [Ladysnake/RATs-Mischief#119]
For some reason whenever I try to run my Minecraft server when I have this mod on & the mialeemisc.json config file has already been generated the server is unable to open.
[server_crash_log.txt](https://github.com/Ladysnake/RATs-Mischief/files/12307351/server_crash_log.txt)

**Comment by Hugo0272** [comment]
I have the same error

**Comment by Lhruggles** [comment]
yeah I also got this crash

**Comment by Wawwior** [comment]
The actual problem here is that the file cant be loaded, seems to me a codec error. Since it doesn't store any valuable information, it can be deleted for each start.

**Comment by LEclipse1001** [comment]
Yeah but it gets really painful deleting the file every single time i restart the server & so i can't have automated server restarts.

**Comment by LEclipse1001** [comment]
Hopefully the mod updates soon then

**Comment by Hugo0272** [comment]
There is an updated version of the library on rat's discord in the #rattiest-ganh channel uploaded by amy, download that if you don't want to wait

**Comment by LEclipse1001** [comment]
Thank You!

**Comment by unilock** [comment]
> There is an updated version of the library on rat's discord in the #rattiest-gang channel uploaded by amy, download that if you don't want to wait

For the untrustworthy, here's a link to the relevant Discord message (you'll need to be in the [Rattiest Gang](https://discord.gg/rattiestgang) guild): https://discord.com/channels/924373275688198204/924373276141162529/1139917255094829156

For the slightly more trustworthy, here's a direct link to just the updated library mod: https://cdn.discordapp.com/attachments/924373276141162529/1139917254750908488/mialeemisc-1.0.38.jar
